src/main/java/com/laborete/LaboreteAPI/exception/ResourceBadRequestException.java

```diff
@@ -0,0 +1,11 @@
+package com.laborete.LaboreteAPI.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
```

Comment:
Please use ControllerAdvice approach.
Create a class that extends `ResponseEntityExceptionHandler` and mark this class with `@ControllerAdvice` attribute (probably it's required to specify `assignableTypes`). And inside of this class define exception handlers - public methods with attribute `@ExceptionHandler`. An example of such method:
```
@ExceptionHandler({AccessDeniedException.class})
    public ResponseEntity<Object> handleAccessDeniedException(AccessDeniedException ex) {
        log.error("Access denied exception {}", ex.getMessage());
        return ResponseEntity.status(HttpStatus.FORBIDDEN).contentType(MediaType.APPLICATION_JSON).body(this.getErrorMessageResponseDto("Forbidden", ex));
    }
```